_episodes/05-memory.md

```diff
@@ -255,14 +275,23 @@ in two different channels is redundant, rather than different.
 
 For our workshops, this means the instructor should say out loud commands as they type them on the screen while engaging learners in [participatory live coding]({{ page.root }}/14-live/).
 
-## Faded Examples
+## Guided Practice at Work
+
```

Comment:
I think we talked about making the exercise in this section optional -- just flagging in case this was an accidental omission and not a decision.